frontend/src/utils/api.js

```diff
@@ -78,7 +78,15 @@ const api = {
       submit: (action, template) => post(`/process-measurements/template_submit/`, form({ action, template })),
     },
   },
-
+  projects: {
+    get: projectId => get(`/projects/${projectId}/`),
+    add: project => post("/projects/", project),
+    update: project => patch(`/projects/${project.id}/`, project),
+    list: (options, abort) => get("/projects", options, { abort }),
+    listExport: options => get("/project/list_export/", {format: "csv", ...options}),
+    summary: () => get("/projects/summary"),
+    search: q => get("/projets/search/", { q }),
```

Comment:
Where is this used? I could be mistaken, but I think that was part of the search viewset.. which is not implemented for projects? (from what I remember that was used for the global searchbar at the top left menu in the interface)

Reply:
Would we want the projects to be searched by the Quick search ? I think if someone type Potatosalad it should point them to the Project if it exists ...

Reply:
I think that's done directly in the table with the table filters.

Reply:
We are talking two different things here. The filters are done automaticallyon the table columns.

We have another functionality which is the jump bar search (general search across the entities of the website). 
![image](https://user-images.githubusercontent.com/1617741/129103835-2944b410-da31-4d4e-ba63-d6779ecdbdc4.png)


What Ulysse meant is that it would be useful to add project search (by name) to that general jump bar. This is done with the search action in the viewset/search endpoint, and some hooking (would have to check exactly how it is done)